### PR TITLE
Add SigmoidRange layer

### PR DESCRIPTION
**Pull request by bibs2091**
Hello,

This is an implementation of the SigmoidRange layer mentioned in https://github.com/tensorflow/tfjs/issues/5452#issue-965182223 .

The SigmoidRange takes two parameters  **min**  and **max** and forces the output to be between them. The default value of **min** and **max** are **0** and **1** respectively (a normal sigmoid) 
The SigmoidRange equation goes this way:
`SigmoidRange(x)  = sigmoid(x) * (max - min) + min`

Please review it and tell me if I should add anything

### CODE REVIEW

**Comment by bibs2091** [comment]
@googlebot I signed it!

**Review by pyu10055 — COMMENTED** [review]
The PR looks good, thanks you for the contribution. 
I did some googling, I did not see this layer is defined by either keras or tf.keras
I am trying to understand the use case, can you provide any links to the definition of this activation layers?
Also if you can add tests would be great, thanks

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/5548#-:-MiN5DOK1F5JtmuA2ZGP:b-30zye4)* status: 0 of 1 approvals obtained

**Comment by bibs2091** [comment]
@pyu10055 Thank you for taking the time to review my PR.
The SigmoidRange layer is inspired from the framework `fastai` which has more than 21k star in GitHub, you can check for the docs of the layer [here ](https://docs.fast.ai/layers.html#SigmoidRange) and the code [here](https://github.com/fastai/fastai/blob/master/fastai/layers.py#L97).

The layer is used when you want to bound the output of your model in a certain range. For my exact use case, I use it for my recommendation system engine, I know for sure that the ratings are between 1 and 5 and my model can accept float values, and SigmoidRange makes a perfect sense to be used. I think other regression tasks can benefit from that if the output min and max are already known (example: when the model task it to output the axis (x,y) of the nose in an image, the SigmoidRange will assure that the coordinates are bigger than 0 and smaller than the size of the image)

One can argue that the model will learn that value is between min and max, but it will take some epochs to learn that. I ran a simple performance test on it, and it really helped accelerating the training speed in the first epochs:
image

Finally, if you think the feature is not essential enough or the name is too confusing, we can rename the layer into Sigmoid (since there's no sigmoid layer) and this layer optionally can receive two params, min and high, if the user does not set them they are automatically set to 0 and 1 respectively making a the normal sigmoid function.

**Review by pyu10055 — COMMENTED** [review]
@bibs2091 Thank you for the detailed explanation. I think keep it as SigmoidRange would be better, since it is different from the official Sigmoid definition.
Please let me when you have the tests ready. Thanks!

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/5548#-:-Mj6P6ji2EaOyzC7WkTl:b4sfycz)* status: 0 of 1 approvals obtained
